libcloudforensics/providers/gcp/internal/compute.py

```diff
@@ -505,6 +505,62 @@ def CreateImageFromDisk(self,
     self.BlockOperation(response)
     return GoogleComputeImage(self.project_id, '', name)
 
+  def CreateImageFromGcsTarGz(
+      self,
+      gcs_uri: str,
+      name: Optional[str] = None) -> 'GoogleComputeImage':
+    """Creates a GCE image from a Gzip compressed Tar archive in GCS.
+
+    Args:
+      gcs_uri (str): Path to the compressed image archive
+          (image.tar.gz) in Cloud Storage. It must be gzip compressed
```

Comment:
"It must be a gzip compressed tar archive, with .tar.gz. extension" ?

Reply:
yes that is true and this is the standard export generated by image export.

Reply:
added "a"